models.go

```diff
@@ -55,6 +55,43 @@ var nip47MethodIcons = map[string]string{
 	NIP_47_LIST_TRANSACTIONS_METHOD: "transactions",
 }
 
+type InfoResponse struct {
```

Comment:
Can you prefix and add TODO comments to all these to move to a different file where we put API response models?